cmd/crc/cmd/start.go

```diff
@@ -63,35 +73,77 @@ func runStart(arguments []string) error {
 
 	client := machine.NewClient()
 	result, err := client.Start(startConfig)
-	if err != nil {
-		return err
+	return &result, err
+}
+
+func renderStartResult(result *machine.StartResult, err error) error {
+	return render(&startResult{
+		Success:       err == nil,
+		Error:         errorMessage(err),
+		ClusterConfig: toClusterConfig(result),
+	}, os.Stdout, outputFormat)
+}
+
+func toClusterConfig(result *machine.StartResult) *clusterConfig {
+	if result == nil || result.Error != "" {
+		return nil
 	}
-	if result.Status != "Running" {
-		return fmt.Errorf("Unexpected status of the OpenShift cluster: %s", result.Status)
+	return &clusterConfig{
+		URL: result.ClusterConfig.ClusterAPI,
+		AdminCredentials: credentials{
+			Username: "kubeadmin",
+			Password: result.ClusterConfig.KubeAdminPass,
+		},
+		DeveloperCredentials: credentials{
+			Username: "developer",
+			Password: "developer",
+		},
```

Comment:
This can be done later, but `machine.ClusterConfig` could probably be adjusted to have the same structure rather than having that logic here.